app/src/main/java/com/mymeetings/android/model/EventViewAlert.kt

```diff
@@ -0,0 +1,17 @@
+package com.mymeetings.android.model
+
+import com.mymeetings.android.model.strategies.ViewAlertType
+import com.mymeetings.android.utils.ClockUtils
+
+class EventViewAlert(
+    timeToShowItAsImmediateEvent: Long?,
+    timeToShowItAsRunningEvent: Long,
+    clockUtils: ClockUtils
```

Comment:
Why is `clockUtils` passed to a model class, is this behaving like a ViewModel?

Reply:
Its a Domain model that deals with ViewAlert of Event. So it has logics. We are going to use ViewModel just for presentation.